circuits/cpp/src/aztec3/circuits/kernel/private/testing_harness.hpp

```diff
@@ -41,11 +44,12 @@ namespace aztec3::circuits::kernel::private_kernel::testing_harness {
 using aztec3::circuits::compute_empty_sibling_path;
 
 // Some helper constants for trees
-constexpr size_t MAX_FUNCTION_LEAVES = 2 << (aztec3::FUNCTION_TREE_HEIGHT - 1);
+constexpr size_t MAX_FUNCTION_LEAVES = 1 << aztec3::FUNCTION_TREE_HEIGHT;  // 2^(height-1)
```

Comment:
Should the comment be `2^height`?

Reply:
@dbanks12 Suyash has a point that 1 << aztec3::FUNCTION_TREE_HEIGHT; is really equal to 2^FUNCTION_TREE_HEIGHT